sdks/java.md

```diff
@@ -7,29 +7,74 @@ Coming Soon
 ### Initialize Bandwidth Client
 
 ```java
-Foo foo = new bar();
+import com.bandwidth.voice.controllers.APIController;
+import com.bandwidth.voice.Configuration;
+import com.bandwidth.voice.Environments;
+
+
+//Set the voice client configuration with credentials
+Configuration voiceConfig = new Configuration().newBuilder()
```

Comment:
I'd rename this variable to just `config` since the way APIMatic's namespacing works is by combining all of the config into 1